From 00d2925528ef26025412263da02cb92e4664db65 Mon Sep 17 00:00:00 2001
From: muzena <trebelnik2@gmail.com>
Date: Wed, 3 Aug 2022 12:21:49 +0200
Subject: [PATCH] Updae patch1

---
 tests/test_xmlparse.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/test_xmlparse.py b/tests/test_xmlparse.py
index 1bf0ebe5..15ca8f75 100644
--- a/tests/test_xmlparse.py
+++ b/tests/test_xmlparse.py
@@ -314,6 +314,8 @@ def testAlterDisk():
     check = _make_checker(disk)
     check("type", "block")
     check("device", "lun")
+    check("sgio", None, "unfiltered")
+    check("rawio", None, "yes")
 
     disk = _get_disk("sda")
     check = _make_checker(disk)
-- 
2.25.1

